app/src/main/kotlin/com/flxrs/dankchat/data/database/UserDisplayEntity.kt

```diff
@@ -0,0 +1,22 @@
+package com.flxrs.dankchat.data.database
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "user_display")
+data class UserDisplayEntity(
+    @PrimaryKey(autoGenerate = true) val id: Int,
+    @ColumnInfo(name = "target_user") val targetUser: String, // target user to apply the color and/or alias
+    @ColumnInfo(name = "enabled") val enabled: Boolean,
+    @ColumnInfo(name = "color_enabled") val colorEnabled: Boolean,
+    @ColumnInfo(name = "color_hex") val color: Int,
```

Comment:
Don't save it as a plain hex but instead save it as a `Color` type and add a custom converter instead of manually doing the conversion